test-packages/test-services-odata-common/generate-common-entity.ts

```diff
@@ -152,7 +154,9 @@ The idea behind this entity is to use only odata-common imports and use it in th
 const imports = `
   import { Moment } from "moment";
   import { BigNumber } from "bignumber.js";
-  import { AllFields, CollectionField, ComplexTypeField, Constructable, ConstructorOrField, CustomField, CustomDeSerializers, defaultDeSerializers, DefaultDeSerializers, DeserializedType, DeSerializers, EdmTypeField, EntityApi, EntityBase as Entity, entityBuilder, EntityBuilderType, Field, FieldBuilder, FieldOptions, mergeDefaultDeSerializersWith, OneToOneLink, OrderableEdmTypeField, PropertyMetadata, Time } from '../src/internal';    
-  import { customTestDeSerializers } from '../../../test-resources/test/test-util';
+  import { AllFields, CollectionField, ComplexTypeField, Constructable, ConstructorOrField, CustomField, CustomDeSerializers, defaultDeSerializers, DefaultDeSerializers, DeserializedType, DeSerializers, EdmTypeField, EntityApi, EntityBase as Entity, entityBuilder, EntityBuilderType, Field, FieldBuilder, FieldOptions, mergeDefaultDeSerializersWith, OneToOneLink, OrderableEdmTypeField, PropertyMetadata, Time } from '../../packages/odata-common/src/internal';
+  import { customTestDeSerializers } from '../../test-resources/test/test-util';
   `;
 const disableEslint = '/* eslint-disable */';
+
+generateCommonEntity();
```

Comment:
Minor: Perhaps you can add a technical debt ticket that we rename this to `generate-common-service` and create also service,api, complextype... files so that this service is more like the other ones - but this is really if we have not something better to do so most likely never.

Reply:
I agree that we may want to improve but a ticket feels unnecessary because it will be super low prio. Also it is quite obvious that this is different and a bit lacking so it wont be hard to find this if you somehow find time to work on it. @marikaner would you prefer I add this tech debt item?